editor/plugins/animation_player_editor_plugin.h

```diff
@@ -202,6 +259,8 @@ class AnimationPlayerEditorPlugin : public EditorPlugin {
 	virtual bool handles(Object *p_object) const;
 	virtual void make_visible(bool p_visible);
 
+	virtual void forward_force_draw_over_viewport(Control *p_overlay) { anim_editor->forward_force_draw_over_viewport(p_overlay); }
```

Comment:
I don't quite understand this. Should there not be one for 2D and one for 3D?

Reply:
For this use case, I designed the API dimension-agnostic, but I can split it easily into 2D and 3D if you find it to make more sense to plugin writers.